content/en/security/code_security/iast/security_controls/_index.md

```diff
@@ -0,0 +1,217 @@
+
+## Configuring security controls
+
+The definition of **Security Controls** shall be placed in the configuration variable **`DD_IAST_SECURITY_CONTROLS_CONFIGURATION`**.
+To configure a list of security controls, follow the format and field specifications below.
+This format uses specific separators to structure each security control entry.
+
+### Format
+
+`<TYPE>:<SECURE_MARKS>:<CLASS/FILE>:<METHOD>:<PARAMETERS (Optional)>:<PARAMETERS TO VALIDATE (Optional)>`
+
+### Field specifications
+| **Field**                             | **Description**                                                                                                                                                                                         |
+|---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Type**                              | Defines the type of control. **Accepted values:** `INPUT_VALIDATOR` or `SANITIZER`.                                                                                                                     |
+| **Secure Marks**                      | List of vulnerability types to apply. Possible values are defined in [Secure marks](#secure-marks). Optionally, use `*` to indicate applicability to all types.                                         |
+| **Class/File**                        | Fully qualified class or file implementing the security control.                                                                                                                                        |
+| **Method**                            | Name of the method implementing the security control.                                                                                                                                                   |
+| **Parameters (Optional)**             | Fully qualified class parameters. Used to distinguish between overloaded methods. If omitted and overloading exists, the security control applies to all overloaded methods.                 |
+| **Parameters to Validate (Optional)** | Zero-based list of parameter positions to validate. The first parameter is position **0**. This field applies **only** to `INPUT_VALIDATOR` types. Used when **not all parameters require validation**. |
+
+
+### Separators
+- **`;` (Semicolon)** → Separates each security control.
+- **`:` (Colon)** → Separates each field within a security control.
+- **`,` (Comma)** → Separates items within a field that accepts a list.
+
+### Secure marks
+
+The following table lists the supported vulnerability types as secure marks for each language:
+
+| **Type** | **Java** | **Node** | **.NET** | **Python** |
+|----------|---------|--------|------|--------|
```

Comment:
I think we should just update the table here: https://github.com/DataDog/documentation/blob/master/content/en/security/code_security/iast/_index.md#overview to include the vuln IDs (separate PR :pray:), and link it from here. Avoiding the mainteinance of these two tables, that will inevitably go out of sync.

Reply:
Done